codec-http/src/main/java/io/netty/handler/codec/http/DefaultHttpHeaders.java

```diff
@@ -78,6 +78,18 @@ public DefaultHttpHeaders() {
         this(true);
     }
 
+    /**
+     * <b>Warning!</b> Setting <code>validate</code> to <code>false</code> will mean that Netty won't
+     * validate that the user supplied header values are malicious.
```

Comment:
are not malicious (like phrased line 91)?

Reply:
```suggestion
     * validate & protect against user-supplied header values that are malicious.
```

You're absolutely right. How does this sound instead ^^

Reply:
LGTM :)